postr/slack_api.py

```diff
@@ -53,4 +93,6 @@ def remove_post(self, post_id: str) -> bool:
 
 if __name__ == '__main__':
     slack = SlackApi()
-    print(slack.post_text('bot started'))
+    print(slack.post_text('Postr has started!'))
+    print(slack.post_photo(url='http://i.imgur.com/FwDiy5m.png', text='schedule'))
```

Comment:
demo code, but I would like the bot to post a welcome message / image on startup later on.